nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/BlobProvider.java

```diff
@@ -200,4 +201,13 @@ default BinaryManager getBinaryManager() {
         return null;
     }
 
+    /**
+     * Checks if the blob provider performs any access control checks before reading a {@link Blob}.
+     *
+     * @param blobInfo the blob information
+     * @return {@code true} if the provider performs security checks before reading a blob, {@code false} otherwise
+     * @since 8.4
+     */
+    boolean performsAccessControl(BlobInfo blobInfo);
```

Comment:
I'd make this a default method and return `false` here to avoid adding it in existing implementations that don't need to return `true`.